src/microbe_stage/editor/PatchMapDrawer.cs

```diff
@@ -1047,6 +1057,13 @@ private void DrawPatchLinks()
     /// </summary>
     private void RebuildMap()
     {
+        playerSpeciesPopulationIndicators = new List<Control>();
```

Comment:
The list should be stored persistently as otherwise this allocates memory unnecessarily each time the map is drawn.